libraries/cli/tsconfig.json

```diff
@@ -5,5 +5,5 @@
     "rootDir": ".",
     "composite": true
   },
-  "include": ["**/*.ts"]
+  "include": ["*.ts"]
```

Comment:
Should we not leave it at the old value in case we ever want to change the structure? What is the downside?

Reply:
No real downside, changing it back.